cmd/skupper/skupper_network.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"github.com/skupperproject/skupper/pkg/utils/formatter"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+func NewCmdNetwork() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "network [command]",
+		Short: "Show information about the sites and services included in the network.",
+	}
+	return cmd
+}
+
+var selectedSite string
+
+func NewCmdNetworkStatus(newClient cobraFunc) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "status",
+		Short:  "Shows information about the current site, and connected sites.",
+		Args:   cobra.MaximumNArgs(1),
+		PreRun: newClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			silenceCobra(cmd)
+
+			sites, err := cli.NetworkStatus()
+
+			if err != nil {
+				fmt.Println("Unable to retrieve network information.")
```

Comment:
It would be helpful to output the error so the user can understand what the issue might be